supplier/views.py

```diff
@@ -0,0 +1,32 @@
+from django.shortcuts import render
+from supplier.models import Supplier
+from .forms import SupplierForm
+
+
+def signup_supplier(request):
+    form = SupplierForm()
+    context = {'form': form, 'message': ['username should be longer than 6',
+                                         'password should be longer than 6',
+                                         'business name should be longer than 6']}
+    if request.method == 'POST':
+        form = SupplierForm(request.POST)
+        user_name = request.POST.get('user_name')
+        password = request.POST.get('password')
+        first_name = request.POST.get('first_name')
+        last_name = request.POST.get('last_name')
+        business_name = request.POST.get('business_name')
+        supplier = Supplier(user_name=user_name,
+                            password=password,
+                            first_name=first_name,
+                            last_name=last_name,
+                            business_name=business_name)
+        if form.is_valid():
+            Supplier.save_supplier(supplier)
```

Comment:
I think we should aspire to have only one registration form for both Supplier and Client. the form will have an option to choose the user type (should not get inserted to the db), and using this we'll gray-out (open the option) the relevant input fields to the specific user type chosen.

Reply:
this will be done in the future at the moment Supplier functionallity will be implemented!

Reply:
Done!